_includes/footer.html

```diff
@@ -36,7 +36,7 @@ <h3>{% t footer.contact_us.title %}</h3>
           <li>Just Arrived</li>
           <li>
             <a href="https://www.google.se/maps/place/Birger+Jarlsgatan+57C,+113+56+Stockholm/@59.3415145,18.0622277,17z/data=!3m1!4b1!4m5!3m4!1s0x465f9d684ea80717:0xe518a0937ab3f86c!8m2!3d59.3415145!4d18.0644164">
```

Comment:
We need to update this link too :)

Reply:
@buren, gotcha!

This used to be a long URL. I put the same short URL on both links. Unsure if this has an effect. Please advice.

Thanks!